exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/OkHttpGrpcExporter.java

```diff
@@ -146,14 +149,36 @@ public void onResponse(Call call, Response response) {
                         ? "gRPC status code " + status
                         : "HTTP status code " + response.code();
                 String errorMessage = grpcMessage(response);
-                logger.log(
-                    Level.WARNING,
-                    "Failed to export "
-                        + type
-                        + "s. Server responded with "
-                        + codeMessage
-                        + ". Error message: "
-                        + errorMessage);
+
+                if (GRPC_STATUS_UNIMPLEMENTED.equals(status)) {
```

Comment:
Noticed this user friendliness that is present in DefaultGrpcExporter was missing